lib/amplitude.dart

```diff
@@ -96,12 +100,18 @@ class Amplitude extends _Amplitude {
     return await _channel.invokeMethod('setServerUrl', jsonEncode(properties));
   }
 
+  Future<void> regenerateDeviceId() async {
+    return await _channel.invokeMethod(
+        'regenerateDeviceId', jsonEncode(_baseProperties()));
+  }
+
   /// Dynamically adjust server URL
   Future<void> setUseDynamicConfig(bool useDynamicConfig) async {
     Map<String, dynamic> properties = _baseProperties();
     properties['useDynamicConfig'] = useDynamicConfig;
 
-    return await _channel.invokeMethod('setUseDynamicConfig', jsonEncode(properties));
+    return await _channel.invokeMethod(
```

Comment:
Try not to mix auto formatting the files together with your feature change. Usually you can create a separate PR for only doing formatting. It's fine to leave it this time.